src/main/kotlin/io/github/inductiveautomation/kindling/log/NameTreePanel.kt

```diff
@@ -0,0 +1,45 @@
+package io.github.inductiveautomation.kindling.log
+
+import com.formdev.flatlaf.extras.FlatSVGIcon
+import io.github.inductiveautomation.kindling.core.FilterChangeListener
+import io.github.inductiveautomation.kindling.core.FilterPanel
+import io.github.inductiveautomation.kindling.utils.Column
+import io.github.inductiveautomation.kindling.utils.getAll
+import net.miginfocom.swing.MigLayout
+import javax.swing.JPanel
+import javax.swing.JPopupMenu
+import javax.swing.JScrollPane
+
+class NameTreePanel(logEvents: List<SystemLogEvent>) : FilterPanel<LogEvent>() {
+    private val tree = LogTree(logEvents)
+
+    private var currentSelectedLeafNodes = tree.selectedLeafNodes.map { it.name }
+
+    override fun filter(item: LogEvent) : Boolean {
+        val results = item.logger in currentSelectedLeafNodes
+        return results
+    }
+
+    override val tabName: String = "Loggers"
+
+    override fun isFilterApplied(): Boolean = true //TODO: Implement this function
```

Comment:
Need to implement `isFilterApplied` to tab highlighting. Forgot to do this. Is this even used anymore since the tabs are just icons now?

Reply:
<img width="138" alt="image" src="https://github.com/inductiveautomation/kindling/assets/39345262/4eeef651-ca18-49da-8991-6acb4ab1cd8e">

It is, yeah - it gives any modified tabs a background color shift to indicate there's a change there